src/std/source.ss

```diff
@@ -1,32 +1,32 @@
 (export #t)
 
-(import (for-syntax ./stxutil)
-        ./sugar)
-
-(begin-for-syntax
+(import (for-syntax :std/stxutil
+                    :std/misc/path
+                    :std/misc/ports)
+        :std/stxutil
```

Comment:
Can we avoid this? It is really intended to be a syntax only import.

Reply:
and it looks completely unecessary too.

Reply:
Done by moving syntax to std/sugar. Thanks.